packages/foreman/foreman/foreman.spec

```diff
@@ -259,7 +263,11 @@ BuildRequires: systemd
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.0.7
+BuildRequires: %{?scl_prefix}npm(@babel/plugin-transform-modules-amd) = 7.9.0
+BuildRequires: %{?scl_prefix}npm(@babel/plugin-transform-modules-commonjs) = 7.9.0
+BuildRequires: %{?scl_prefix}npm(@babel/plugin-transform-modules-systemjs) = 7.9.0
+BuildRequires: %{?scl_prefix}npm(@babel/preset-env) = 7.9.5
```

Comment:
These aren't really required, they are provided by foreman-js and in the package-json just to pin the version used by npm